modules/ask/src/main/AskEmbed.scala

```diff
@@ -0,0 +1,189 @@
+package lila.ask
```

Comment:
now this file is scary. "Frozen magic" seem like correct terminology. Has plenty of logic, with number constants, hacky regexes. I would feel much better if it was not in the project.